Nucleon/resources/logs/Nucleon_Logs.txt

```diff
@@ -0,0 +1 @@
+2021-03-31 07:25:52,934 | MainThread | INFO | push_app_api, push_app_api_v2 functions are not available
```

Comment:
this file should contain the sample logs (without sensitive information) for Nucleon so that we can upload the file and test the dashboard. Can you please provide the sample logs?

Reply:
where can I find this log file (sample log)?
inside the directory Nucleon-> resources->logs I have one file named "Nucleon_Logs.txt" with one line:
`2021-03-31 07:25:52,934 | MainThread | INFO | push_app_api, push_app_api_v2 functions are not available`
and I have a directory named "testlogs" with the blow info to include this file?
`2021-04-14 07:26:49,759 | MainThread | INFO | push_app_api, push_app_api_v2 functions are not available
2021-04-14 07:26:49,768 | MainThread | INFO | Starting Tests for manifestfile: Nucleon/Nucleon.manifest.json appfile: Nucleon/Nucleon.json
2021-04-14 07:26:49,829 | MainThread | WARNING | These tests do not cover multiple metadata(_sourceCategories,_source etc). They may or may not be replaced
2021-04-14 07:26:49,830 | MainThread | DEBUG | test_all_params_replaced_in_dashboards (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:49,832 | MainThread | DEBUG | test_folder_description (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:49,879 | MainThread | INFO | ImageName: Nucleon.png Width: 72 Height: 72
2021-04-14 07:26:49,889 | MainThread | INFO | ImageName: Nucleon_Overview.png Width: 1911 Height: 1039
2021-04-14 07:26:49,895 | MainThread | INFO | ImageName: Nucleon_Threats_By_Source_Country_MAP.png Width: 1903 Height: 711
2021-04-14 07:26:49,897 | MainThread | DEBUG | test_folder_structure (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS   
2021-04-14 07:26:49,899 | MainThread | DEBUG | test_has_app_description (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:49,900 | MainThread | DEBUG | test_has_category (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS       
2021-04-14 07:26:49,902 | MainThread | DEBUG | test_has_community_url (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS        
2021-04-14 07:26:49,904 | MainThread | DEBUG | test_has_dashboard_description (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,270 | MainThread | DEBUG | test_has_helpdoc_url (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,272 | MainThread | DEBUG | test_has_lookup_operator_from_external_files (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,274 | MainThread | DEBUG | test_has_save_operator (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,275 | MainThread | DEBUG | test_has_scheduled_searches (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,276 | MainThread | DEBUG | test_has_valid_dashboard_title (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,952 | MainThread | DEBUG | test_has_valid_icon (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,954 | MainThread | DEBUG | test_not_using_default_lookup (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,956 | MainThread | DEBUG | test_valid_timerange (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,957 | MainThread | DEBUG | test_version_preview_consistent (sumoapputils.common.testapp.TestClassicDashboardsV2) ... PASS
2021-04-14 07:26:50,958 | MainThread | INFO | Ran 16 tests in 1.130s

2021-04-14 07:26:50,960 | MainThread | INFO | APPNAME: Nucleon | TOTAL TESTCASES: 16 | RESULT: ALL TESTS PASSED
2021-04-14 07:26:50,963 | MainThread | INFO | ==================================================================================================================================`

Reply:
nops not this log file I mean sample log messages or metrics that you used for developing these dashboards in sumo logic portal. Am assuming you'd have ingested some data into sumo and developed your dashboards using that data right?

Reply:
OK, I understand, thanks!
this is a sample log file:
[activethreats_data.txt](https://github.com/SumoLogic/sumologic-public-partner-apps/files/6309443/activethreats_data.txt)

Reply:
Thanks I see there are some ips there please make sure it doesn't have any sensitive info since this is a public repo. Also you can remove the .log file and commit this file instead.

Reply:
OK, we checked it, it doesn't have any sensitive information.
we removed the log file

Reply:
Thanks please remove this file as well Nucleon/resources/logs/Nucleon_Logs.txt Also remove testlogs directory and move the activethreats_data.txt to Nucleon/resources/logs/Nucleon_Logs.txt. Sample logs needs to be in Nucleon/resources/logs/ as per instructions in the doc.

Reply:
It's done